.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Would be good to remove local vscode workspace setting. Thanks

Reply:
Really sorry about this… I completely forgot about the .vscode directory.